pkg/amazon/sdk/sdk.go

```diff
@@ -194,6 +193,60 @@ func (s sdk) CreateStack(ctx context.Context, name string, template *cf.Template
 	return err
 }
 
+func (s sdk) CreateChangeSet(ctx context.Context, name string, template *cf.Template, parameters map[string]string) (string, error) {
+	logrus.Debug("Create CloudFormation Changeset")
+	json, err := template.JSON()
+	if err != nil {
+		return "", err
+	}
+
+	param := []*cloudformation.Parameter{}
+	for name := range parameters {
+		param = append(param, &cloudformation.Parameter{
+			ParameterKey:     aws.String(name),
+			UsePreviousValue: aws.Bool(true),
+		})
+	}
+
+	update := fmt.Sprintf("Update%s", time.Now().Format("2006-01-02-15-04-05"))
```

Comment:
Will this work across timezones? I think using `Now` and then formatting like you have will drop the timezone.

```suggestion
	update := fmt.Sprintf("Update-%s", time.Now().Format("2006-01-02-15-04-05"))
```

Reply:
this string is not actually visible to end user: once applied, the changeset is deleted by cloudformation. This is more a unique ID. Could use uuid.generate() assuming there's a common golang lib for this purpose